cartographi/dataloaders/scalar/era5_max_wave_height.py

```diff
@@ -0,0 +1,37 @@
+from cartographi.dataloaders.scalar.abstract_scalar import ScalarDataLoader
+
+import xarray as xr
+
+
+class ERA5MaxWaveHeightDataLoader(ScalarDataLoader):
+    def import_data(self, bounds):
+        """
+        Reads in data from an ERA5 NetCDF file.
+        Renames coordinates to 'lat' and 'long'
+
+        Args:
+            bounds (Boundary): Initial boundary to limit the dataset to
+
+        Returns:
+            xr.Dataset:
+                ERA5 wave dataset within limits of bounds.
+                Dataset has coordinates 'lat', 'long', and variable 'hmax'
+        """
+        # Open Dataset
+        if len(self.files) == 1:    data = xr.open_dataset(self.files[0])
+        else:                       data = xr.open_mfdataset(self.files)
+        # Change column names
+        data = data.rename({'latitude': 'lat',
+                            'longitude': 'long'})
+        # Change domain of dataset from [0:360) to [-180:180)
+        data = data.assign_coords(long=((data.long + 180) % 360) - 180)
```

Comment:
I feel like we discussed this already in person when I first added the existing wave height dataloader? It seems to work in the examples I've run so far and with the wave variables the issue you describe ought to be pretty obvious since I would expect you would get a good chunk of the NaN values on land shifted into the ocean. I'll try putting something together with the raw data to confirm it matches what we see in the mesh anyway.

Reply:
My connection isn't good enough to post screenshots right now but I just had a look at the raw data from 01/12/2022 in QGIS and a mesh I built for that day and all the different wave data types seem to match up as well as expected.

Reply:
There is data in the usual place on the SAN for November and December 2022 if you want to try running an example yourself to confirm.